lonboard/geopandas.py

```diff
@@ -0,0 +1,296 @@
+from typing import Any, Dict, List, Optional, Union
+
+import geopandas as gpd
+import numpy as np
+import pandas as pd
+
+from . import Map, basemap, viz
+from .colormap import apply_categorical_cmap, apply_continuous_cmap
+
+__all__ = ["LonboardAccessor"]
+
+
+@pd.api.extensions.register_dataframe_accessor("lb")
+class LonboardAccessor:
+    def __init__(self, pandas_obj):
+        self._validate(pandas_obj)
+        self._obj = pandas_obj
+
+    @staticmethod
+    def _validate(obj):
+        if not isinstance(obj, gpd.GeoDataFrame):
+            raise AttributeError("must be a geodataframe")
+
+    def explore(
+        self,
+        column: Optional[str] = None,
+        cmap: Optional[str] = None,
+        scheme: Optional[str] = None,
+        k: Optional[int] = 6,
+        categorical: bool = False,
+        elevation: Union[str, np.ndarray] = None,
+        extruded: bool = False,
+        elevation_scale: Optional[float] = 1,
+        alpha: Optional[float] = 1,
+        layer_kwargs: Optional[Dict[str, Any]] = None,
+        map_kwargs: Optional[Dict[str, Any]] = None,
+        classification_kwds: Optional[Dict[str, Any]] = None,
+        nan_color: Optional[Union[List[int], np.ndarray[int]]] = [255, 255, 255, 255],
+        color: Optional[str] = None,
+        vmin: Optional[float] = None,
+        vmax: Optional[float] = None,
+        wireframe: bool = False,
+        tiles: Optional[str] = None,
+        highlight: bool = False,
+        m: Optional[Map] = None,
```

Comment:
A personal preference here would be to move these before any of the kwargs dicts.

Reply:
Well, that is a good point that the docs below refer to these as keyword args, but in actuality all of the above parameters are positional arguments (there's no `*` in the signature). I'd be inclined to match the geopandas signature as much as possible, but then make any _added_ parameters be keyword-only.